things/api/things/http/transport.go

```diff
@@ -52,6 +52,13 @@ func MakeHandler(tracer opentracing.Tracer, svc things.Service) http.Handler {
 		opts...,
 	))
 
+	r.Post("/things/bulkCreate", kithttp.NewServer(
```

Comment:
Using `/things/bulk` is clear enough, IMHO. Also, you can consider adding a query param such as `bulk=true` to the existing endpoint for single Thing creation (especially since it already is in the plural - `/things`). The same goes for Channels.

Reply:
I mentioned something about combining the endpoints in #484 but I proposed the single endpoint use the bulk endpoint with only 1 thing in the array.  The single endpoint currently accepts 1 JSON object rather than an array so this would need to be changed as well.